features/percy/provider.feature

```diff
@@ -1,4 +1,4 @@
-@percy @javascript @selenium
```

Comment:
Why did you remove `@percy` tag?
This is used against Percy

Reply:
Right, my bad 👍 It was my mistake 👍

Reply:
@Martouta